desktop/api/src/main/java/org/datacleaner/util/WidgetUtils.java

```diff
@@ -655,4 +655,21 @@ public static void setDefaultButtonStyle(AbstractButton b) {
             b.setUI(DefaultButtonUI.get());
         }
     }
-}
\ No newline at end of file
+    
+    public static void disableComponent(JComponent component) {
```

Comment:
This seems very unrelated? And not really the way I would prefer disabling to work because it is ultimately a ComponentUI responsibility. Background colors etc. to use for disabled/enabled will vary a lot depending on where the component is.